packages/shared/trade/src/transaction-confirmation/components/portfolio-compare.tsx

```diff
@@ -90,6 +106,23 @@ export const PortfolioCompare = ({
         }
         columns={columns}
       />
+      <LinkText
+        sx={{
+          textAlign: 'right',
+          marginTop: theme.spacing(2),
+          cursor: 'pointer',
+        }}
+        onClick={handleCopy}
+      >
+        <FormattedMessage defaultMessage={'Copy Transaction Details'} />
+        <CopyIcon
+          fill={theme.palette.typography.accent}
+          sx={{
+            marginLeft: '0.25rem',
```

Comment:
Can we switch these to theme.spacing and px? Trying to destroy references to rems

Reply:
Sounds good